solutions/09-11/creditCard.html

```diff
@@ -3,32 +3,36 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-     <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css">
+    <link rel="stylesheet" type="text/css" href="style.css">
     <title>Document</title>
 </head>
 <body>
     <div class="container">
         <div class="center">
-            <form>
+            <form id="myForm">
                 <div class="form-group owner">
                     <label for="owner">Owner</label>
                     <input type="text" class="form-control" id="owner">
+                    <span id="errOwner"></span>
                 </div>
                 <div class="form-group" id="card-number-field">
-                    <label for="cardNumber">Card Number</label>
-                    <input type="text" class="form-control" id="cardNumber">
+                    <label for="cardNumber">Card Number<i class="fa" id="icon"></i></label>
+                    <input type="text" class="form-control" id="cardNumber" value="">
+                    <span id="errCard"></span>
                 </div>
                 <div class="validation form-group">
-                    <div class="expirtion-date" id="expiration-date">
+                    <div class="expiration-date" id="expiration-date">
                         <label>Expiration Date</label>
-                        <input type="text" class="form-control" id="exp-date">
+                        <input type="text" class="form-control" id="exp-date" placeholder="MM/YY">
+                        <span id="errDate"></span>
                     </div>
                     <div class="cvv" id="cvv-number">
                         <label>CVV</label>
                         <input type="text" class="form-control" id="cvv">
```

Comment:
Type number, with min and max value can be help full

Reply:
I forgot about it, it was little tricky to display span if input:invalid :)